src/components/structures/SearchBox.js

```diff
@@ -21,7 +21,7 @@ import { _t } from '../../languageHandler';
 import { KeyCode } from '../../Keyboard';
 import sdk from '../../index';
 import dis from '../../dispatcher';
-import rate_limited_func from '../../ratelimitedfunc';
+import { debounce } from 'lodash';
```

Comment:
if we're importing lodash, can we use the latest version please? There's security issues related to releases older than a few days ago which don't affect us but would still be good to patch against.

Reply:
My `package-lock.json` states `4.17.11` which appears to be [the latest version](https://www.npmjs.com/package/lodash), unless they moved to a different package name I'm unaware of?

Reply:
I'm probably just blind and/or dumb - lgtm!